src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ReportsTest.kt

```diff
@@ -368,6 +390,7 @@ class Cas2ReportsTest : IntegrationTestBase() {
           newStatus = event1Details.newStatus.name,
           updatedAt = event1Details.updatedAt.toString().split(".").first(),
           updatedBy = event1Details.updatedBy.username,
+          statusDetails = event1Details.newStatus.statusDetails.joinToString("|") { it.name },
```

Comment:
i appreciate you're following the already established testing pattern here for assertions, but I think it would be really helpful if this had the hardcoded expectation as a string

Reply:
Thanks David - I've changed this [here](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1576/commits/cb3775f8635e96e88063f80f606be4cdb0caa2bd).

Ok to approve?  Thanks.